### DISCUSSION TITLE
Unknown Pack

### DISCUSSION THREAD

**Discussion by SamGamerYT** [Kas-tle/java2bedrock.sh#1562] [Converter Help]
After converting pack this shows up https://gyazo.com/ce19733f8583d66a9c2e71840ea98db7

**Comment by Kas-tle** [comment]
Hello. Please link to the conversion run for this pack.

**Reply by SamGamerYT** [reply]
> Hello. Please link to the conversion run for this pack.

https://github.com/Kas-tle/java2bedrock.sh/issues/1563
